config/environments/development.rb

```diff
@@ -76,4 +76,6 @@
     SwaggerUiEngine::SwaggerDocsController.send(:include, Api::V1::Concerns::CspHeaderBypass)
     SwaggerUiEngine::SwaggerDocsController.send(:layout, 'swagger_ui_engine/layouts/swagger')
   end
+
+  config.force_ssl = true
```

Comment:
We will need to test this to make sure that it does not conflict with Dockerized development via nginx/Passenger.

Reply:
> Looks good - I'd just like a quick check that adding `config.force_ssl` in `config/environments/development.rb` will not impact Dockerized development.

Yes, I think all the local https changes (thin in the gemfile, requiring ssl, etc...) should not be part of this branch

Reply:
I've incorporated recent updates here that do away with `thin` (and without which this doesn't work locally).